docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1419,5 +1387,78 @@ specify a subset of columns to analyzed with the optional ``columns`` property::
 
 This query collects statistics for columns ``col_1`` and ``col_2``.
 
-Note that if statistics were previously collected for all columns, they need to be dropped
+Note that if statistics were previously collected for all columns, they must be dropped
 using :ref:`drop_extended_stats <drop-extended-stats>` command before re-analyzing.
+
+.. _iceberg-table-redirection:
+
+Table redirection
+^^^^^^^^^^^^^^^^^
+
+.. include:: table-redirection.fragment
+
+The connector supports redirection from Iceberg tables to Hive tables
+with the ``iceberg.hive-catalog-name`` catalog configuration property.
+
+File formats
+------------
+
+The following file types and formats are supported for the Iceberg connector:
+
+* ORC
```

Comment:
Really .. all of these are supported in Iceberg .. I thought for data files just ORC and Parquet